lib/packages/core/lib/src/views/home/home_feed.dart

```diff
@@ -55,7 +55,7 @@ class _LMChatHomeScreenState extends State<LMChatHomeScreen> {
   @override
   void initState() {
     Bloc.observer = LMChatBlocObserver();
-    user = LMChatPreferences.instance.getCurrentUser;
+    user = LMChatLocalPreference.instance.getUser()!;
```

Comment:
Can we add some other check for nullability? Or throw an error here.

Reply:
i will add the check in `getUser()` method itself to avoid code duplication